### PR TITLE
Update Day Calculations to not count pauses AFTER decision_date

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: services/apps/alcs/src/providers/typeorm/migrations/1716848951501-update_day_calculations.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+              MAX("date_received_all_items") AS date_received_all_items,
+              max("end_date") AS end_date,
+              max("start_date") AS start_date,
+              bool_or("end_date" IS NULL) AS has_open,
+              count("notice_of_intent_meeting"."uuid") AS meeting_count
+            FROM
+              "alcs"."notice_of_intent"
+            LEFT JOIN "alcs"."notice_of_intent_meeting" ON 
+              "notice_of_intent"."uuid" = "notice_of_intent_meeting"."notice_of_intent_uuid"
+              AND start_date <= COALESCE("decision_date", NOW())
```
Comment (dhaselhan) [inline services/apps/alcs/src/providers/typeorm/migrations/1716848951501-update_day_calculations.ts:104]: This line was changed, adding AND start_date <= COALESCE("decision_date", NOW())

File: services/apps/alcs/src/providers/typeorm/migrations/1716848951501-update_day_calculations.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                AND NOT EXISTS (
+                SELECT
+                    *
+                FROM
+                alcs.application_paused t2
+                WHERE
+                COALESCE(t1.end_date, NOW()) >= t2.start_date
+                AND COALESCE(t1.end_date, NOW()) < COALESCE(t2.end_date, NOW()) --- use coalesce since end_date may be null
+                AND t1.application_uuid = t2.application_uuid)
+            LEFT JOIN alcs.application ON application.uuid = s1.application_uuid
```
Comment (dhaselhan) [inline services/apps/alcs/src/providers/typeorm/migrations/1716848951501-update_day_calculations.ts:41]: This join was added

File: services/apps/alcs/src/providers/typeorm/migrations/1716848951501-update_day_calculations.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                SELECT
+                    *
+                FROM
+                alcs.application_paused s2
+                WHERE
+                    s1.start_date > s2.start_date
+                    AND s1.start_date <= COALESCE(s2.end_date, NOW())
+                    AND s1.application_uuid = s2.application_uuid)
+                AND s1.application_uuid = ANY (p_ids)
+                AND s1.start_date <= COALESCE(application.decision_date, NOW())
```
Comment (dhaselhan) [inline services/apps/alcs/src/providers/typeorm/migrations/1716848951501-update_day_calculations.ts:53]: And this AND here

**Review by mhuseinov — APPROVED** [review]
👍

**Pull request merged**